### PR TITLE
feat:CO-67 CO-69: style leak bugs

### PR DESCRIPTION
**Pull request by arjun-at-roam**
## PR Area

- [x] Frontend

## Type of change

_What kind of change does this PR introduce?_

- [x] Bugfix

## Description

* use reset.scss to (you guessed it) reset the styles.

* PbTech - border and background styling leaks (card border and background are messed up)
  * Problem
image
image
image
image
  * Solution
image
image
image
image

* Samsung - font-weight styling leaks (bold title stuff aren't showing bold enough)
  * Problem
image
image
image
image
  * Solution
image
image
image
image

## Security & risks

- [ ] Adds new third party libraries
- [ ] Adds new `.env` variables
- [ ] Integrates with Third-party apis (dependency must be resilient)
- [ ] Has specific [OWASP security concerns](https://owasp.org/www-project-top-ten/) - list them: (also see [Fixing OWASP Top 10](https://nodegoat.herokuapp.com/tutorial))

## Pre-publish checklist

- [x] I have assigned myself to the ticket
- [x] I have performed a self-review of my code
- [ ] I have performed relevant browser testing
- [ ] I have performed keyboard testing of new or modified interactive components

## Reviewer checklist

- [ ] I have run the code (if applicable)
- [ ] I have reviewed and run any tests (if applicable)

### CODE REVIEW

File: cypress/page-objects/merchantProductPage.ts [hunk 1]
```diff
@@ -3,7 +3,8 @@ class _merchantProductPage {
     url: string,
     targetElement: string,
     position: string,
-    country: 'nz' | 'au' = 'nz'
+    country: 'nz' | 'au' = 'nz',
+    darkMode = false
```
Comment (arjun-at-roam) [inline cypress/page-objects/merchantProductPage.ts:7]: this is for future tests allowing the widget to be injected in dark mode

File: src/styles/reset.scss [hunk 2]
```diff
@@ -9,5 +9,13 @@
...
+    border: none;
+    background: none;
+  }
+  p {
+    margin: none;
+  }
+  strong {
+    font-weight: bold;
   }
 }
```
Comment (arjun-at-roam) [inline src/styles/reset.scss:18]: `reset.scss` is imported into `Theme` provider that wraps around the widget.
You can checkout the Theme provider 👇 
https://github.com/shophumm/humm-widgets-cards-online/blob/feat/CO-67-69-style-leak-bugs/src/providers/Theme.vue

File: src/styles/reset.scss [hunk 3]
```diff
@@ -9,5 +9,13 @@
   dd {
     line-height: normal;
     font-weight: normal;
+    border: none;
+    background: none;
+  }
+  p {
+    margin: none;
```
Comment (ghost) [inline src/styles/reset.scss]: `margin: none;` is not valid css. Use `auto`, `inherit` or `0` instead

**Review by cynthia-tan-roam — APPROVED** [review]
Cypress changes look ok to me :)

**Review by ghost — APPROVED** [review]
Excellent work🚀

File: src/styles/reset.scss (hunk 3 above)
Comment (arjun-at-roam) [reply src/styles/reset.scss]: Turns out the problem I was trying to fix here was not a styling leak. The component was missing margin-top.
This is the issue I thought was a styling leak:
image

Fixed 👇 
image

Comment (ghost) [reply src/styles/reset.scss]: Makes sense – good catch!

**Pull request merged**